project/plugins.sbt

```diff
@@ -1,9 +1,9 @@
-resolvers += Resolver.url("sbt-plugin-releases", new URL("http://scalasbt.artifactoryonline.com/scalasbt/sbt-plugin-releases/"))(Resolver.ivyStylePatterns)
-
 addSbtPlugin("com.github.gseitz" % "sbt-release"     % "1.0.0")
```

Comment:
we'll need to bump this as well - that'll get covered by the cherry-pick, I guess.